documentation/internal/Filters-for-Applications.md

```diff
@@ -0,0 +1,292 @@
+## Filters for Applications
+
+---
+
+### Overview
+
+---
+
+The list of applications to display for the current user is based on:
+
+- its permissions to view a certain group of applications
+- the application type (aka template code)
+- other optional filters selected
+
+The user has access to links to see each type of application list from the top menu or in their dashboard - the link will set basic filters in the main URL route of the list: `/applications`. For example the list to see the Application of type "Drug Registration" would take the user to `/applications?type=user-registration&user-role=applicant`.
+
+The `type` is the template code for the applications type, also displayed in the header of the Applications list.
+The `user-role` is used to build how the columns and rows should be displayed. The user-role is deduced from the permissions that the current user have to view this application type. In this example the user has **applicant** role by what was set in their permissions.
+
+### Example: [UI Design] Applicant - applications list
+
+![Applicant-List](images/Filters-for-Applications-Applicant-List.png)
+
+---
+
+### Filters
+
+---
+
+#### Label filters:
+
+##### type
+
+**This is a compulsory filter**
+In case none is included in URL will be deduced from 1st user's persmissions
+
+Example: `type=user-registration`
+Considered in implementation: **Yes** [#131 - Headers]
+Can have combined values: **No**
```

Comment:
Done! Well spotted.

Reply:
Cool. I think it looks a little neater if these blocks just have line breaks (rather than paragraph breaks), but not a big deal either way. Up to you.
```
Example: `type=user-registration`  (2 spaces at line end forces a line break)  
Considered in implementation: **Yes** [#131 - Headers]  
Can have combined values: **No**  
Options:
- `template-code` - Of application type in list (and header)
```